docs/time_series_performance_benchmarks.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>Should we mention the dataset used here? Does that impact the number of time points which can be returned in under 30 seconds?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/titiler-cmr/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>